ethpm_cli/manifest.py

```diff
@@ -468,3 +468,76 @@ def parse_bool_flag(question: str) -> bool:
         else:
             cli_logger.info(f"Invalid response: {response}.")
             continue
+
+
+def examine_manifest(manifest_path: Path) -> None:
+    raw_manifest = json.loads(manifest_path.read_text())
+    validate_manifest_against_schema(raw_manifest)
+    manifest = ManifestDisplay(raw_manifest)
+    cli_logger.info(f"Package Name: {manifest.package_name}")
+    cli_logger.info(f"Package Version: {manifest.package_version}")
+    cli_logger.info(f"Manifest Version: {manifest.manifest_version}\n")
+    cli_logger.info(f"Sources: \n{''.join(manifest.sources())}")
+    cli_logger.info(f"Contract Types: \n{''.join(manifest.contract_types())}")
+    cli_logger.info(f"Deployments: \n{''.join(manifest.deployments())}")
+    cli_logger.info(f"Build Dependencies: \n{''.join(manifest.build_dependencies())}")
+
+
+class ManifestDisplay:
+    def __init__(self, manifest: Manifest) -> None:
+        self.manifest = manifest
+
+    @property
+    def package_name(self) -> str:
+        return self.manifest["package_name"]
+
+    @property
+    def package_version(self) -> str:
+        return self.manifest["version"]
+
+    @property
+    def manifest_version(self) -> str:
+        return self.manifest["manifest_version"]
+
+    @to_list
+    def sources(self) -> Iterable[str]:
+        if "sources" not in self.manifest:
+            yield "No sources found.\n"
```

Comment:
Maybe just `N/A` or `None`.  The current message kind of sounds like they might be there and just be hidden somewhere..  Same with the other ones that use the same verbage.